src/main/kotlin/no/nav/familie/ef/iverksett/vedtakstatistikk/VedtakstatistikkService.kt

```diff
@@ -1,11 +1,87 @@
 package no.nav.familie.ef.iverksett.vedtakstatistikk
 
+import no.nav.familie.ef.iverksett.domene.Periodebeløp
+import no.nav.familie.ef.iverksett.domene.Vilkårsresultat
+import no.nav.familie.ef.iverksett.infrastruktur.json.AndeltilkjentYtelseJson
+import no.nav.familie.ef.iverksett.infrastruktur.json.IverksettJson
+import no.nav.familie.ef.iverksett.infrastruktur.json.PersonJson
+import no.nav.familie.ef.iverksett.infrastruktur.json.VilkårsvurderingJson
+import no.nav.familie.ef.iverksett.økonomi.tilKlassifisering
+import no.nav.familie.eksterne.kontrakter.ef.Aktivitetskrav
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingDVH
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingResultat
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingType
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingÅrsak
+import no.nav.familie.eksterne.kontrakter.ef.Inntekt
+import no.nav.familie.eksterne.kontrakter.ef.Inntektstype
+import no.nav.familie.eksterne.kontrakter.ef.PeriodeBeløp
+import no.nav.familie.eksterne.kontrakter.ef.Periodetype
+import no.nav.familie.eksterne.kontrakter.ef.Person
+import no.nav.familie.eksterne.kontrakter.ef.Utbetaling
+import no.nav.familie.eksterne.kontrakter.ef.Utbetalingsdetalj
+import no.nav.familie.eksterne.kontrakter.ef.Vedtak
+import no.nav.familie.eksterne.kontrakter.ef.Vilkår
+import no.nav.familie.eksterne.kontrakter.ef.Vilkårsvurdering
 import org.springframework.stereotype.Service
+import java.time.ZoneId
 
 @Service
 class VedtakstatistikkService(val vedtakstatistikkKafkaProducer: VedtakstatistikkKafkaProducer) {
 
-    fun sendTilKafka(vedtakStatistikk: String) {
-        vedtakstatistikkKafkaProducer.sendVedtak(vedtakStatistikk)
+    fun sendTilKafka(iverksettJson: IverksettJson) {
+        val vedtakstatistikk = hentBehandlingDVH(iverksettJson)
+        vedtakstatistikkKafkaProducer.sendVedtak(vedtakstatistikk)
+    }
+
+    private fun hentBehandlingDVH(iverksettJson: IverksettJson): BehandlingDVH {
+        return BehandlingDVH(fagsakId = iverksettJson.fagsakId,
+                             saksnummer = iverksettJson.saksnummer,
+                             behandlingId = iverksettJson.behandlingId,
+                             relatertBehandlingId = iverksettJson.relatertBehandlingId,
+                             kode6eller7 = iverksettJson.kode6eller7,
+                             tidspunktVedtak = iverksettJson.tidspunktVedtak?.atStartOfDay(ZoneId.of("Europe/Paris")), //?
```

Comment:
DVH trenger ZonedDateTime, men vi får vedtaksdato som LocalDate fra sak

Reply:
Men betyr det at de bruker "klokkeslettet" til noe fornuftig? Isåfall må vi kanskje lagre vedtakstidspunkt og ikke vedtaksdato i saksbehandlingsløsningen? @nav-jorn-ola-birkeland

Reply:
Hvis vi bare hare dato, så får vi levere dato. Dvs uten klokkeslett.

Reply:
@nav-jorn-ola-birkeland Kan vi endre kontrakt fra ZonedDateTime til Localdate ? 
Eller burde vi sende dato med atStartOfDay() og beholde ZonedDateTime i kontrakter?